static/utils.py

```
@@ -0,0 +1,87 @@
+def query_name(name):
+    # TODO
+    found = lxml.etree.parse(io.BytesIO(USERS_XML.encode())).xpath(".//user[name/text()='%s']" % name)
+    content = "<b>Surname:</b> %s%s" % (found[-1].find("surname").text if found else "-", HTML_POSTFIX)
+    return content
+
+
+def query_include(v):
+    # TODO
+    backup = sys.stdout
+    sys.stdout = io.StringIO()
+    program = (open(v, "rb") if not "://" in v else urllib.request.urlopen(v)).read()
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)